docs/DESIGN_VISION.md

```
@@ -1,257 +1,54 @@
-│        │  Back of Head │            │
-│        │  (brown hair) │            │
-│        └───────┬───────┘            │
-│          ┌─────┴─────┐             │
-│          │ Shoulders  │             │
-│          │ (t-shirt)  │             │
-│          └─────┬─────┘             │
-│    ┌───┬───┬───┬───┐               │
-│    │F1 │F2 │F3 │F4 │  keyboard    │
-│    └───┴───┴───┴───┘               │
-└─────────────────────────────────────┘
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)